adf.mf.resource.ADFInfoBundle_fr={MSG_ATTEMPT_TO_GO_NEGATIVE_NESTING:"Tentative d''obtention d''une valeur n\u00e9gative dans le niveau d''imbrication pour {0} {1}",MSG_LOGIN_TENANT_NAME:"Nom du locataire",MSG_LOGIN_PASSWORD:"Mot de passe",MSG_LOGIN_USERNAME:"Nom utilisateur",MSG_LOGIN_WELCOME:"Bienvenue",MSG_ERROR_IN_SCRIPT:"Erreur JavaScript \u00e0 l''\u00e9tape {0}. Erreur : {1}",MSG_ERROR_IN_LOGOUT:"Erreur lors de la tentative de d\u00e9connexion. Erreur : {0}","oracle.core.ojdl.logging.MessageIdKeyResourceBundle":"",
MSG_ERROR_IN_LOGIN:"Erreur lors de la tentative de connexion. Erreur : {0}",MSG_ERROR_REGISTER_ACTIVATION_HANDLER:"Erreur lors de l''inscription du gestionnaire d''activation. Code d''erreur : {0}",MSG_LOGIN_TENANT_ID:"ID de locataire",LBL_WARNING_DISPLAY_STR:"Avertissement",MSG_ERROR_TREENODEITERATOR_UPDATE_KEYS:"probl\u00e8me lors de la mise \u00e0 jour des cl\u00e9s {0} - {1}",LBL_ERROR_DISPLAY_STR:"Erreur",MSG_ERROR_MNGD_BEANS_NOT_SUPPORTED:"Les beans g\u00e9r\u00e9s dans un environnement autre que Java ne sont pas pris en charge.",
LBL_FATAL_DISPLAY_STR:"Fatal",MSG_UNABLE_TO_FETCH_SET:"impossible d''extraire l''ensemble - {0}",MSG_ERROR_TREENODEITERATOR_UPDATE_PROVIDERS:"impossible de mettre \u00e0 jour le fournisseur car cm n'est pas d\u00e9fini.",LBL_OK_DISPLAY_STR:"OK",MSG_ERROR_NOTIFYING_DATA_CHANGE_LISTENER:"Un probl\u00e8me est survenu lors de la notification des processus d''\u00e9coute de modification des donn\u00e9es pour {0}",MSG_ERROR_PROCESSING_PROVIDER_CHANGES:"Erreur dans processDataChangeEvent lors du traitement des modifications de fournisseur {0}",
MSG_ERROR_PURGING_CACHE:"Erreur lors de la tentative de purge du cache : {0}",MSG_ERROR_UPDATING_CACHE:"Un probl\u00e8me est survenu lors de la mise \u00e0 jour du cache suite \u00e0 un \u00e9v\u00e9nement de modification des donn\u00e9es.",MSG_ADFMOBILE_NOT_FOUND:"Echec d''ADFmf sur la commande suivante {0}:{1} car la classe ADFMobile est introuvable.",MSG_QUERY_PARAMETER_NOT_FOUND:"param\u00e8tre de requ\u00eate ''{0}'' introuvable.",MSG_EL_PARSER_UNEXPECTED_UNIARY_OP_FOUND:"op\u00e9ration inattendue (unaire) d\u00e9tect\u00e9e",
MSG_EL_PARSER_UNEXPECTED_BINARY_OP_FOUND:"op\u00e9ration inattendue (binaire) d\u00e9tect\u00e9e",MSG_EL_PARSER_NESTED_EL_NOT_SUPPORTED:"expressions EL imbriqu\u00e9es non prises en charge",MSG_EL_PARSER_UNEXPECTED_OPERATOR_FOUND:"op\u00e9rateur inattendu d\u00e9tect\u00e9",LBL_CONFIRMATION_DISPLAY_STR:"Confirmation",MSG_EL_PARSER_UNEXPECTED_TERNARY_OP_FOUND:"op\u00e9ration inattendue (ternaire) d\u00e9tect\u00e9e",LBL_INFO_DISPLAY_STR:"Infos",MSG_ERROR_IN_ERROR_CALLBACK:"Erreur d\u00e9tect\u00e9e lors de l'appel du callback d'erreur",
MSG_ERROR_IN_INDIVIDUAL_NOTIFICATION_CALLBACK:"Erreur d\u00e9tect\u00e9e lors de l''appel du callback de notification de modification des donn\u00e9es pour {0}. Erreur : {1}",MSG_ERROR_IN_BULK_NOTIFICATION_CALLBACK:"Erreur d\u00e9tect\u00e9e lors de l''appel du callback de notification de modification des donn\u00e9es en masse. Erreur : {0}",MSG_EL_PARSER_UNEXPECTED_STRING_FOUND:"constante de cha\u00eene inattendue d\u00e9tect\u00e9e",MSG_ERROR_PROCESSING_VAR_CHANGES:"Erreur dans processDataChangeEvent lors du traitement des modifications de variable {0}",
MSG_EL_PARSER_UNEXPECTED_VARIABLE_FOUND:"variable inattendue d\u00e9tect\u00e9e",MSG_SET_LOCAL_VALUE_FAILED:"\u00e9chec de la d\u00e9finition de la valeur locale, appel en cours des callbacks \u00e9chou\u00e9s \u00e0 propos de {0}",MSG_ERROR_RESETTING_BINDING_CONTEXT:"Erreur lors de la r\u00e9initialisation du contexte de liaison - {0}",MSG_EL_PARSER_UNEXPECTED_NUMBER_CONSTANT_FOUND:"constante num\u00e9rique inattendue d\u00e9tect\u00e9e",MSG_ERROR_CALLING_ERROR_HANDLERS:"Erreur d\u00e9tect\u00e9e lors de l''appel des gestionnaires d''erreur - {0}",
MSG_ERROR_PROCESSING_REMOVE_DATA_CHANGE:"Un probl\u00e8me est survenu lors du traitement de la modification des donn\u00e9es (enlever) pour #''{{0}.collectionModel''} - {1}",MSG_ERROR_PROCESSING_CREATE_DATA_CHANGE:"Un probl\u00e8me est survenu lors du traitement de la modification des donn\u00e9es (cr\u00e9er) pour #''{{0}.collectionModel''} - {1}",MSG_ERROR_PROCESSING_UPDATE_DATA_CHANGE:"Un probl\u00e8me est survenu lors du traitement de la modification des donn\u00e9es (mettre \u00e0 jour) pour #''{{0}.collectionModel''} - {1}",
MSG_LOGIN_WARNING:"Erreur : vous devez entrer le nom utilisateur et le mot de passe.",MSG_ERROR_IN_EL_PARSING:"Erreur d''analyse EL : {0} -- [colonne {1}] {2}",MSG_EL_PARSER_UNABLE_TO_RESOLVE:"Impossible de r\u00e9soudre {0} \u00e0 l''aide des variables donn\u00e9es : ",MSG_EL_PARSER_INCORRECT_OPERANDS:"nombre incorrect d''op\u00e9randes {0} - {1}",MSG_EL_PARSER_MISMATCH:"non-correspondance avec () ou []",MSG_EL_PARSER_MISSING_ENDING:"marqueur de fin d'expression introuvable",MSG_EL_PARSER_UNKNOWN_CHAR_FOUND:"caract\u00e8re inconnu ''{0}'' d\u00e9tect\u00e9",
MSG_EL_PARSER_UNEXPECTED_CONSTANT_FOUND:"constante inattendue d\u00e9tect\u00e9e",MSG_ERROR_IN_REQUEST:"Demande : {0}, exception : {1}",MSG_SKIP_REMOTE_WRITE:"Java n'\u00e9tant pas disponible, l'\u00e9criture distante sera ignor\u00e9e.",MSG_GETVALUE_FAILED_CB_ERROR:"obtenir la valeur du {0}e callback \u00e9chou\u00e9 en erreur - {1}",MSG_GETVALUE_SUCCESS_CB_ERROR:"obtenir la valeur du {0}e callback de succ\u00e8s en erreur - {1}",MSG_EXCEPTION:"Exception : {0}",MSG_LOGIN_LOGIN_BUTTON_LABEL:"Connexion",
MSG_LOGIN_EXIT_BUTTON_LABEL:"Quitter l'application",MSG_EXCEPTION_RESOLVING:"Exception : {0} lors de la tentative de r\u00e9solution de : {1}",MSG_SET_LOCAL_VALUE_FAILED_CB:"le callback \u00e9chou\u00e9 de la valeur locale d\u00e9finie a \u00e9t\u00e9 ex\u00e9cut\u00e9 mais g\u00e9n\u00e8re - {0}",MSG_SET_LOCAL_VALUE_SUCCESS_CB_FAILED:"\u00e9chec du callback de succ\u00e8s de la valeur locale d\u00e9finie ({0}e) avec l''exception {1}",MSG_ERROR_PROCESSING_DATA_CHANGE_EVENT:"erreur lors du traitement de l''\u00e9v\u00e9nement de modification des donn\u00e9es - {0}",
MSG_UNABLE_TO_SET_DATA_CHANGE_VALS:"Impossible de d\u00e9finir les valeurs de l''\u00e9v\u00e9nement de modification des donn\u00e9es [{0}]"};